requirements.txt

```diff
@@ -0,0 +1,8 @@
+xmldiff
+yamlpath
+json2html
+types-openpyxl
+types-PyYAML
```

Comment:
Looks like we'll need:

- PyYAML
- jinja2
- pandas (although this is from the tool I'm writing, so I can add it there if this lands first)
- tables
- openpyxl
- pygments

Reply:
Ok - good deal. Thanks.

Reply:
Don't be shy and add them as well - if you use `pip install` on a system where part of the requirements are already satisfied, they are simply skipped.